connector/spanmetricsconnector/connector_test.go

```diff
@@ -1998,13 +1999,18 @@ func TestConnectorWithCardinalityLimit(t *testing.T) {
 
 	assert.NoError(t, connector.ConsumeTraces(context.Background(), traces))
 
-	metrics := connector.buildMetrics()
+	// first buildMetrics emit zero datapoint value
+	_ = connector.buildMetrics()
```

Comment:
I think it would be a good idea to check here that we are getting the metrics we expect with value 0

Reply:
This PR (https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/39052) introduces a feature where call metrics are initialized with a value of 0. However, I haven't added a unit test to verify this behavior. I plan to add a separate test function to handle it.

In the current test case, I am just skip the first 0 value.